### PR TITLE
Notify_author_question template changed

### PR DESCRIPTION
**Pull request by Gauravano**
solves #1713 
I have changed the UI according to the issue discussion 
@jywarren  kindly suggest if any changes required. Thank You.

### CODE REVIEW

File: app/views/answer_mailer/notify_question_author.html.erb [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
+<p><%= @answer.content.truncate(30) %></p>
 
-<p>https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>#a<%= @answer.id %></p>
+Do NOT reply to this email; click this link to see the answer:
 
-<p><b>Here is the answer that <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @answer.author.name %>"><%= @answer.author.name %></a> wrote:</b></p>
-
-<hr /> 
-
-<p><%= raw auto_link(RDiscount.new(@answer.body_email(ActionMailer::Base.default_url_options[:host])).to_html) %></p>
```
Comment (jywarren) [inline app/views/answer_mailer/notify_question_author.html.erb]: I think we're not quite ready to remove the full text from the email -- maybe we could revert this and think that through as a follow-up? I'd like to keep the entire text in for now. I'm not as worried about spammers, and think it's really great for a "quick look" at a new post. Thanks!

File: app/views/answer_mailer/notify_question_author.html.erb [hunk 2]
```diff
@@ -1,23 +1,18 @@
-Hi! A new answer has been posted for your question '<a href='https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>'><%= @answer.node.title %></a>'. Do NOT reply to this email; click this link to see the answer:
+Hi! <a href='https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @answer.author.name %>'><%= @answer.author.name %></a> responded :
```
Comment (jywarren) [inline app/views/answer_mailer/notify_question_author.html.erb]: Cool!

**Comment by Gauravano** [comment]
hi @jywarren, I have implemented changes as requested by you, sir. Now the whole answer would be visible but only initial part of question would be visible on the subject of mail sent to users. 

pull2_att2

pull2_attt2

**Comment by jywarren** [comment]
Great! @pdhixenbaugh - what do you think about this? The screenshot here shows the new subject line. 

@Gauravano - I think @pdhixenbaugh also noted we could change the prefix from [PublicLab] to [PL] -- what do you think of that? 

Sorry for my slow response, we just had the Barnraising (our annual conference) and then I've been traveling since so I'm just catching up now! Your work on this is much appreciated :-)

**Comment by jywarren** [comment]
Oh, actually it was `[PLab]` and not `[PL]`. I think that makes sense!

**Comment by Gauravano** [comment]
First of all, no problem about delay @jywarren sir.
As you suggested I have changed [PublicLab] to [PLab], also I want to highlight that apart from the subject, content part of mail is also different.Here's the updated template, kindly review.Thank you.

screenshot from 2017-11-14 16-31-24

**Comment by jywarren** [comment]
Awesome, this looks great. Just going to wait one more day to see if @pdhixenbaugh has a chance to chime in, thanks!!!

**Comment by pdhixenbaugh** [comment]
Looks beautiful!

**Comment by jywarren** [comment]
Awesome, thanks to everybody for pitching in on this one. We're making solid progress!

**Pull request merged**